columnflow/plotting/plot_all.py

```diff
@@ -231,38 +254,51 @@ def draw_profile(
         "histtype": "step",
     }
     defaults.update(kwargs)
+    if "yerr" not in defaults:
+        if h.storage_type.accumulator is not hist.accumulators.WeightedSum:
+            raise TypeError(
+                "Error bars calculation only implemented for histograms with storage type WeightedSum "
+                "either change the Histogram storage_type or set yerr manually",
+            )
+        defaults["yerr"] = calculate_stat_error(h, error_type)
     h.plot1d(**defaults)
 
 
 def draw_errorbars(
     ax: plt.Axes,
     h: hist.Hist,
     norm: float | Sequence | np.ndarray = 1.0,
+    error_type: str = "variance",
```

Comment:
Since we mainly use this method to draw the data errorbars, using the corresponding `error_type` as default might be preferable

```suggestion
    error_type: str = "poisson_unweighted",
```